src/xcode/ENA/ENA/Source/Workers/Store/SecureStore.swift

```diff
@@ -22,12 +22,21 @@ import Foundation
 /// The `SecureStore` class implements the `Store` protocol that defines all required storage attributes.
 /// It uses an SQLite Database that still needs to be encrypted
 final class SecureStore: Store {
+
+	// MARK: - Static.
```

Comment:
That's not static as `static` but rather `immutable`.

Reply:
removed mark